File: chapter-02/exercise-05-simple-math/SimpleMath.java
class SimpleMath {
  public static void main(String[] args) {
    var input = System.console().readLine("What is the first number? ");
    var firstNumber = Integer.parseInt(input);

    input = System.console().readLine("What is the second number? ");
    var secondNumber = Integer.parseInt(input);

    var sum = firstNumber + secondNumber;
    var difference = firstNumber - secondNumber;
    var product = firstNumber * secondNumber;
    var quotient = firstNumber / secondNumber;

    System.out.printf("%1$d + %2$d = %3$d%n%1$d - %2$d = %4$d%n%1$d * %2$d = %5$d%n%1$d / %2$d = %6$d%n",
      firstNumber, secondNumber, sum, difference, product, quotient);
  }
}

File: exercise-05-simple-math/simple_math.py
first_number = int(input("What is the first number? "))
second_number = int(input("What is the second number? "))

summation = first_number + second_number
difference = first_number - second_number
product = first_number * second_number
quotient = first_number / second_number

print(f"{first_number} + {second_number} = {summation}\n" \
      f"{first_number} - {second_number} = {difference}\n" \
      f"{first_number} * {second_number} = {product}\n" \
      f"{first_number} / {second_number} = {int(quotient)}")
